src/server/conn_context.cc

```diff
@@ -15,6 +16,20 @@ namespace dfly {
 
 using namespace std;
 
+StoredCmd::StoredCmd(const CommandId* d, CmdArgList args) : descr(d) {
```

Comment:
Non functional change - moved this logic from main service so all storedcmd args manipulation is in storedcmd